### PR TITLE
Scene understanding - cleaned up version (no bonus commits) of 8369

### PR DESCRIPTION
**Pull request by davidkline-ms**
This is a fresh version of #8369 that removes the already merged commits that were confusing GitHub.

### CODE REVIEW

File: Assets/MRTK/Providers/Experimental/WindowsMixedReality/SceneUnderstanding/.gitignore [hunk 1]
```diff
@@ -1,5 +0,0 @@
-# ========================= #
-# MSBuildForUnity generated #
-# ========================= #
-.obj
-.bin
```
Comment (MaxWang-MS) [inline Assets/MRTK/Providers/Experimental/WindowsMixedReality/SceneUnderstanding/.gitignore:5]: Just want to make sure we really wish to remove these as they seem reasonable

File: Assets/MRTK/Providers/Experimental/WindowsSceneUnderstanding/WindowsSceneUnderstandingObserver.cs [hunk 2]
```diff
@@ -337,11 +464,16 @@ public bool TryFindCentermostPlacement(Guid quadGuid, Vector2 objExtents, out Ve
...
+#endif // SCENE_UNDERSTANDING_PRESENT
         }
 
-        #endregion Public Methods
+#endregion Public Methods
+
+#if SCENE_UNDERSTANDING_PRESENT
 
-        #region Private
+#region Private
```
Comment (MaxWang-MS) [inline Assets/MRTK/Providers/Experimental/WindowsSceneUnderstanding/WindowsSceneUnderstandingObserver.cs]: Just want to confirm: you don't like the ident here right?

File: Assets/MRTK/Providers/WindowsMixedReality/XR2018/WindowsMixedRealitySpatialMeshObserver.cs [hunk 3]
```diff
@@ -232,7 +232,7 @@ public override void Update()
         public override void Resume()
         {
 #if UNITY_WSA
-            if (WaitingForSceneObserverAccess)
+            if (IsRunning)
```
Comment (MaxWang-MS) [inline Assets/MRTK/Providers/WindowsMixedReality/XR2018/WindowsMixedRealitySpatialMeshObserver.cs:235]: Do you mind explaining the change here?

File: Assets/MRTK/Providers/Experimental/WindowsMixedReality/SceneUnderstanding/.gitignore (hunk 1 above)
Comment (davidkline-ms) [reply Assets/MRTK/Providers/Experimental/WindowsMixedReality/SceneUnderstanding/.gitignore:5]: i did not intend to remove those

File: Assets/MRTK/Providers/WindowsMixedReality/XR2018/WindowsMixedRealitySpatialMeshObserver.cs (hunk 3 above)
Comment (davidkline-ms) [reply Assets/MRTK/Providers/WindowsMixedReality/XR2018/WindowsMixedRealitySpatialMeshObserver.cs:235]: this was restoring the existing observer to the original code. it should not have been modified in the scene understanding branch

File: Assets/MRTK/Providers/Experimental/WindowsMixedReality/SceneUnderstanding/.gitignore (hunk 1 above)
Comment (davidkline-ms) [reply Assets/MRTK/Providers/Experimental/WindowsMixedReality/SceneUnderstanding/.gitignore:5]: i will add them to the repo's .gitignore file

**Pull request merged**